.github/workflows/test.yml

```
@@ -12,7 +17,7 @@ jobs:
 
     strategy:
       matrix:
-        ideaVersion: [ "2024.2.0.1" ]
+        ideaVersion: [ "2025.1.3", "2024.1.7" ]
```

Could other OTP versions be added here? 
```suggestion
        ideaVersion: [ "2025.1.3", "2024.1.7" ]
        otp-version: ["24.3.4.6", "28.0.1"]
```

---

This change will be reverted - to bump OTP/Elixir requires lexer changes, which is fine, but requires changes.

One big change at a time. 😬